src/components/Organizations/Manage/index.js

```diff
@@ -15,6 +16,11 @@ import PaginatedTable from '../PaginatedTable/PaginatedTable';
 import { AddButtonContent, Footer, RemoveModalHeader, RemoveModalParagraph, TableActions, Tag } from '../SharedStyles';
 import { StyledUsers } from '../Users/Styles';
 
+export const userTypes = ['viewer', 'owner', 'admin'];
```

Comment:
```suggestion
export const userTypes = ['viewer', 'admin', 'owner'];
```

I'd reorder this (assuming the order is as presented) to preserve hierarchy (and try to reinforce that owner is a higher level of permission than admin?